frontend/src/pages/Application.tsx

```diff
@@ -11,6 +11,7 @@ export type ApplicationProps = {
 
 export const Application: React.FC<ApplicationProps> = ({ path }: ApplicationProps) => {
   const [pageNum, setPageNum] = useState<0 | 1 | 2 | 3 | 4 | 5>(0);
+  const [isCompleteInOneSittingModalOpen, setIsCompleteInOneSittingModalOpen] = useState(true);
 
   const next = () => {
     if (pageNum < 5) {
```

Comment:
You can also trigger the Confirm Submission Modal by checking is pageNum is in state 5.